src/components/SelectionCart/SelectionCart.js

```diff
@@ -52,13 +55,32 @@ class SelectionCart extends Component {
             'reservation': [
             ]
         };
+        let reservedComponents = {
+            'memberID': this.userID,
+            'activeComponents': [
+            ]
+        }
         for(let id in addedItems) {
             data.reservation.push({
                 'componentID' : id,
                 'quantity' : addedItems[id].quantity,
             })
+            reservedComponents.activeComponents.push({
+                'componentID' : id,
+                'quantity' : addedItems[id].quantity,
+            })
         }
         MockReservation.reservations.push(data);
+        const reserveIdx = ActiveComponents.reservations.findIndex(item => item.memberID === this.userID);
```

Comment:
Can this be changed to a more descriptive name?
```suggestion
        const reservationIndex = ActiveComponents.reservations.findIndex(item => item.memberID === this.userID);
```